### ISSUE TITLE
Trouble installing on HP Chromebook 14

### ISSUE THREAD

**Issue by YrrchSebor** [dnschneid/crouton#1110]
Trying to install the default, Precise with XFCE, and everything seems fine until it hangs at 'Processing triggers for libc-bin...  ldconfig deferred processing now taking place'. It seems to just stop there.. I keep trying to scroll down, since the page doesn't seem to scroll on it's own, usually i can scroll up and then down to see where it's at, but it just won't go past this line...

**Comment by jflheureux** [comment]
I just had the same problem. Thanks to @YrrchSebor for the return key trick.

Here's my output from when it hanged and what happened after:

Processing triggers for libc-bin ...
ldconfig deferred processing now taking place
Please specify a username for the primary user: 
Please specify a username for the primary user: jflheureux
Enter new UNIX password: 
Retype new UNIX password: 
Sorry, passwords do not match
passwd: Authentication token manipulation error
passwd: password unchanged
Enter new UNIX password: 
Retype new UNIX password: 
passwd: password updated successfully


Basically, the console got stuck on the ldconfig line and never displayed the username question. In reality, it was waiting for my input. When I hit the return key, I saw the username question. But when typing my user name, I didn't saw the characters on the screen. I saw them only when hitting return again.

If you run into this problem, simply enter your username, hit return, enter your password, hit return, confirm your password, and hit return a last time. Then you will be displayed the success message. Your current shell will forever hide the current line. Close it and open a new one.

**Comment by YrrchSebor** [comment]
Ah, sorry for the late reply. By any chance did you change your font size or zoom settings in Chrome OS? I eventually suspected that changing font size or zoom was the cause if this issue?

On Sat, Aug 6, 2016 at 7:20 AM, jflheureux <notifications@github.com<mailto:notifications@github.com>> wrote:

I just had the same problem. Thanks to @YrrchSebor<https://github.com/YrrchSebor> for the return key trick.

Here's my output from when it hanged and what happened after:

Processing triggers for libc-bin ...
ldconfig deferred processing now taking place
Please specify a username for the primary user:
Please specify a username for the primary user: jflheureux
Enter new UNIX password:
Retype new UNIX password:
Sorry, passwords do not match
passwd: Authentication token manipulation error
passwd: password unchanged
Enter new UNIX password:
Retype new UNIX password:
passwd: password updated successfully

Basically, the console got stuck on the ldconfig line and never displayed the username question. In reality, it was waiting for my input. When I hit the return key, I saw the username question. But when typing my user name, I didn't saw the characters on the screen. I saw them only when hitting return again.

If you run into this problem, simply enter your username, hit return, enter your password, hit return, confirm your password, and hit return a last time. Then you will be displayed the success message. Your current shell will forever hide the current line. Close it and open a new one.

**Comment by jflheureux** [comment]
It happened to me in another context while running a command that outputted a lot of lines. I think the problem is when the console has too many messages in history, it starts to display the current line just outside of the view.

**Comment by lvlaser** [comment]
Posting this to help save time for others who get confused, this has a simple solution.

Same thing happened to me, and actually hung me up for 30 minutes or more because it didn't occur to me that I wasn't seeing the last line of text.  

**Try zooming out** on just the console window with **ctrl -** and you will see all lines.

The confusing version:
![screenshot 2016-09-18 at 10 35 48](https://cloud.githubusercontent.com/assets/13839671/18616571/46ef8944-7d8c-11e6-857d-716abed8099b.png)

The version after I zoom out one level on that page:
![screenshot 2016-09-18 at 10 38 32](https://cloud.githubusercontent.com/assets/13839671/18616589/8b4c00cc-7d8c-11e6-9a95-021ea3336517.png)
